templates/typescript/cache_ttl.ts

```diff
@@ -0,0 +1,26 @@
+async function handleRequest(request) {
```

Comment:
Any reason for sometimes having an explicit return type and sometimes not?

Reply:
Nope, just a little sloppy; I'll address that.